build-conf/README.md

```diff
@@ -292,6 +292,16 @@ zunit_reportDatasets | Comma separated list of 'report' type data sets
 zunit_reportOptions | BPXWDYN creation options for creating 'report' type data sets
 zunit_dependenciesDatasetMapping | DBB property mapping to map dependencies to different target datasets
 
+### zCEE3.Properties
```

Comment:
@M-DLB can you please align to the standard language prefix, which is the base name of the language script in lower case?

Reply:
Done